.github/workflows/deploy_docs_and_viewer.yml

```diff
@@ -0,0 +1,26 @@
+name: deploy docs and viewer
+on:
+    workflow_dispatch:
+        inputs:
+          args:
+            description: 'Arguments to the remote script'
+            required: false
+            type: string
+#   push:
+#     branches:
+#       - master
+#       - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+    #   - run: cd docs
+    #   - run: pwd
+      - run: pip install mkdocs-material
+      - run: git submodule update --init --recursive
+    #   &?
+      - run: mkdocs gh-deploy --config-file docs/mkdocs.yml
```

Comment:
This needs to specify a sub-folder, so it deploys to `/molstar-volseg/docs` while the viewer is directly at `/molstar-volseg`